src/lib/application.ts

```diff
@@ -107,6 +107,12 @@ export class Application extends ChildableComponent<
      */
     static readonly EVENT_BOOTSTRAP_END = ApplicationEvents.BOOTSTRAP_END;
 
+    /**
+     * Emitted when validation is being run.
+     * The listener will be given an instance of {@link ProjectReflection}.
+     */
+    static readonly EVENT_VALIDATION_RUN = ApplicationEvents.VALIDATION_RUN;
```

Comment:
I think `validateProject` might be better - it immediately implies that the callback will be passed a project, thoughts?

Reply:
So `ApplicationEvents.VALIDATE_PROJECT`?

Reply:
Yep, and `validateProject` for the string, otherwise lgtm

Reply:
Thanks all set up :)